handlers_racing_test.go

```diff
@@ -163,10 +163,16 @@ func TestSignaturePass(t *testing.T) {
 		for j, response := range responses {
 			switch response.Type {
 			case contentsignature.Type:
-				err = verifyContentSignatureResponse(
-					testcase.signaturerequests[j].Input,
-					response,
-					testcase.endpoint)
+				rawInput, decodeErr := base64.StdEncoding.DecodeString(testcase.signaturerequests[j].Input)
+				if decodeErr != nil {
+					t.Fatalf("in test case %d on endpoint %q, error '%v' in response %d;\nrequest was: %+v\nresponse was: %+v failed to decode input",
+						i, testcase.endpoint, decodeErr, j, testcase.signaturerequests[j], response)
+				}
+				if req.URL.RequestURI() == "/sign/hash" {
```

Comment:
ugh, okay, we def got to delete this type after this

Reply:
If you are referring to the fact that the signature is generated differently in `SignHash()` and `SignData()` - it looks like the same situation exists for `contentsignaturepki`, but it's just that we have no tests covering that scenario.